src/util/sss_python.h

```diff
@@ -16,17 +16,25 @@
 
 #if PY_MAJOR_VERSION >= 3
 #define IS_PY3K
-#define MODINITERROR return NULL
+
+#define MODINITERROR(module)                        \
+    Py_XDECREF(module);                             \
+    return NULL
```

Comment:
Please wrap in `do { } while(0)` (as for example done in `TYPE_READY`)
(for explanation see for example http://c-faq.com/cpp/multistmt.html)